### ISSUE TITLE
Should this repo be archived?

### ISSUE THREAD

**Issue by pwright** [syndesisio/syndesis-documentation#29]
Should this repo be archived? or is there any use associated with it?

**Comment by TovaCohen** [comment]
Yes, probably it should be archived. We are planning to reorganize the doc directory in the main syndesis repo in the next month or so. As part of that work, I think I might like to copy the style guide from this repo, syndesis/syndesis-documentation, to that directory.  Then I'll make sure there is nothing else useful in syndesis/syndesis-documentation, and get it archived. Thanks for bringing this to my attention, Paul.

**Comment by TovaCohen** [comment]
@heiko-braun can we archive the syndesis-documentation repo?
I looked through all the content. There is only out-of-date information or empty stub files. 
I copied some of the style guide content for creating asciidoc to an asciidoc-tips file in the syndesis doc directory. 
I believe that archiving this repo just makes it read-only. 
To archive it, it looks like someone just needs to click "Archive this Repository" at the bottom of this page: https://github.com/syndesisio/syndesis-documentation/settings
WDYT?
